pkg/proxy/options/options.go

```diff
@@ -56,6 +57,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.MappingFile, "mapping-file", o.MappingFile, "Config file mapping paths to backends")
 	fs.StringVar(&o.RootDirectory, "root-directory", o.RootDirectory, "Root directory.")
 	fs.StringVar(&o.RootKubeconfig, "root-kubeconfig", o.RootKubeconfig, "The path to the kubeconfig of the root shard.")
+	fs.StringVar(&o.ShardsKubeconfig, "shards-kubeconfig", o.ShardsKubeconfig, "The path to the kubeconfig used for communication with other shards.")
```

Comment:
nit - AFAICS these credentials are used to create ClusterWorkspace informers for all shards (including root), so I'd s/other/all here 

It's still a little confusing given we also require root-kubeconfig, but perhaps that can be deprecated/removed in a future iteration

Reply:
yeah, i'm not sure. In general we need a privileged credentials for making wildcard requests against all shards. In addition to that we also need access to the root cluster for identities.

Reply:
Not blocking on it, but as previously mentioned I'd adjust the description slightly e.g:

`The path to the kubeconfig used for communication with all shards.`

Reply:
sorry, fixed :)